### PR TITLE
feat: toripunks dapp

### PR DESCRIPTION
**Pull request by omniwired**
preview
https://deploy-preview-554--teritori-dapp.netlify.app/dapp/tori-punks/lottery
image

### CODE REVIEW

**Comment by zxxma** [comment]
LGTM (quick UX review only)

File: packages/screens/DAppStore/apps/toripunks/components/buttonLabel/BuyToripunksButton.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+export const BuyToripunksButton: React.FC<{ size: "S" | "Mobile" }> = ({
+  size,
+}) => (
+  <ButtonLabel
+    text="BUY TORIPUNKS"
+    size={size}
+    actionable
+    onPress={() => {
+      Linking.openURL(
+        "https://app.teritori.com/collection/tori-tori1plr28ztj64a47a32lw7tdae8vluzm2lm7nqk364r4ws50rgwyzgsapzezt"
```
Comment (hthieu1110) [inline packages/screens/DAppStore/apps/toripunks/components/buttonLabel/BuyToripunksButton.tsx:14]: Maybe we should put this hardcode value to a config/settings file or at least a constant

File: packages/screens/DAppStore/apps/toripunks/content/ComicBooks.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            height: 180,
+          }}
+        >
+          <ButtonLabel
+            text="BUY COMIC BOOKS"
+            size="S"
+            actionable
+            onPress={() => {
+              Linking.openURL(
+                "https://app.teritori.com/collection/tori-tori1syx6q5rxhntng93pmk9xep25638dqy3dcnppvvytnccxrp7urt7s2570jc"
```
Comment (hthieu1110) [inline packages/screens/DAppStore/apps/toripunks/content/ComicBooks.tsx:88]: Put this to config/settings files

**Review by hthieu1110 — DISMISSED** [review]
LGMT

**Review by sakul-budhathoki — APPROVED** [review]

File: packages/screens/DAppStore/apps/toripunks/components/buttonLabel/BuyToripunksButton.tsx (hunk 1 above)
Comment (n0izn0iz) [reply packages/screens/DAppStore/apps/toripunks/components/buttonLabel/BuyToripunksButton.tsx:14]: this should at least be a relative link

File: packages/screens/DAppStore/apps/toripunks/content/ComicBooks.tsx (hunk 2 above)
Comment (omniwired) [reply packages/screens/DAppStore/apps/toripunks/content/ComicBooks.tsx]: fixed as relative

File: packages/screens/DAppStore/apps/toripunks/components/buttonLabel/BuyToripunksButton.tsx (hunk 1 above)
Comment (omniwired) [reply packages/screens/DAppStore/apps/toripunks/components/buttonLabel/BuyToripunksButton.tsx]: fixed as relative

**Pull request merged**